spec/hardware/iommu.fmf

```diff
@@ -0,0 +1,26 @@
+summary: |
+    Select or provision a guest with the `Input–output memory management unit`.
+
+description: |
+    .. code-block::
+
+       iommu:
+           # Boolean, whether the guest supports IOMMU
+           is-supported: "x"
+           # String, the model name for virtualizated IOMMU
+           model: "virtio"|"intel"|"smmuv3"
+
+    .. versionadded:: 1.35
+
+example:
+  - |
+    # Require a guest that supports IOMMU
+    iommu:
+        is-supported: true|false
```

Comment:
I would expect `true` instead of `true|false` - if the example is showing me "this is how to request a guest that supports IOMMU", `false` would tell me how to request one that does *not* support it.

Reply:
It is a typo here. Updated